### DISCUSSION TITLE
[Feature Request] enhance alerting to lookback, trend, and possibly future predictions

### DISCUSSION THREAD

**Discussion by Sam-H101** [grafana/grafana#63830] [Alerting]
From using other alerting tools, a few examples that standout would be:

numerical alerting:
- look back n number of values and if its a steady increase / decrease alert. (the last n number of values are all increasing or decreasing)
- if the trend of the last time interval is going down at a fast rate, alert
- in the next hour is this value going to be over x. 

non numerical alerting:
- if the last value is "string" alert or if that "string" has appeared n number of times over a time frame

This would be beneficial as we would not need to configure multiple alerting tools to achieve the desired results. It would be time saving if we could have all of the alerts in a single system.

**Comment by pritster5** [comment]
Just curious if there is any workaround you know of that currently exists to alert based on the trend of the last n values?

**Reply by Sam-H101** [reply]
At the moment I have been using zabbix alerting for the trends, and feed the grafana alerts into zabbix using a custom web hook I wrote.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82802. Feel free to continue the discussion around this there. Thank you!
